packages/auth/src/authClients/supabase.ts

```diff
@@ -32,13 +46,39 @@ export const supabase = (client: Supabase): AuthClientSupabase => {
   return {
     type: 'supabase',
     client,
-    login: ({ email, password }) => client.auth.signIn({ email, password }),
-    logout: () => client.auth.signOut(),
-    signup: ({ email, password }) => client.auth.signUp({ email, password }),
+    login: async ({ email, password, provider }) => {
```

Comment:
Can't you just pass all parameters on to supabase, and let them try to figure out what kind of auth is requested? No need for us to try to duplicate their logic.

Reply:
I imagine one could, but I chose to enumerate the options to self document in code what is required of each flavor of auth. 

And it ensures that only the values needed for each is sent -- and matches the https://supabase.io/docs/client/auth-signin examples.

Hope that makes sense.